src/oss/confluent.sh

```diff
@@ -941,28 +1005,50 @@ start_command() {
     validate_java_version "${1}"
     set_or_get_current
     echo "Using CONFLUENT_CURRENT: ${confluent_current}"
-    start_or_stop_service "start" "services" "${@}"
-    status=$?
-    if [[ ${status} -eq 0 ]]; then
-        is_enterprise && start_or_stop_service "start" "enterprise_services" "${@}"
-    fi
+    start_or_stop_service "start" "${@}"
 }
 
 stop_command() {
-    set_or_get_current
-    echo "Using CONFLUENT_CURRENT: ${confluent_current}"
-    is_enterprise
-    status=$?
-    if [[ ${status} -eq 0 ]]; then
-        start_or_stop_service "stop" "rev_enterprise_services" "${@}"
-        status=$?
+    start_or_stop_service "stop" "${@}"
+}
+
+start_or_stop_service() {
+    local start_or_stop="${1}"
+
+    local service="${2}"  # e.g. "kafka" or ""
+
+    if [[ -n "${service}" ]]; then
+        ! service_exists "${service}" && die "Unknown service: ${service}"
+        local dep_var_name=`echo "deps_${start_or_stop}_${service}" | sed "s/-/_/"`
```

Comment:
why do we need to `sed`? Why not naming correctly the `deps_` version ?

Reply:
The variable name may not have any "-" in it, but the service command the user passes will have hyphens.

Reply:
@kkonstantine ^

Reply:
My point was, why the function doesn't map exactly the service name?

Reply:
In other words, is there any reason why `deps_${start_or_stop}_${service}` is not sufficient on its own?

Reply:
Because service names have "-" in them (e.g. kafka-rest), and variable names can't afaik

Reply:
Wasn't sure if that's used as a function or as a variable. Fine then.